epub33/rs/index.html

```diff
@@ -172,6 +172,13 @@ <h2>EPUB Publications</h2>
 									href="https://www.w3.org/TR/epub-33/#sec-resource-locations">Resource Locations</a>
 								[[!EPUB-33]].</p>
 						</li>
+						<li>
+							<p id="confreq-rs-data-urls">It MUST prevent attemps at top-level navigation to data URLs
```

Comment:
Is there a more precise term for this? Or is `top-level navigation` defined somewhere we can reference?

Reply:
Can we consider any top-level content document a [top-level browsing context](https://html.spec.whatwg.org/multipage/browsers.html#top-level-browsing-context)?

If so, we don't need to separate the concepts and can use only the HTML term.

I wasn't sure if they're considered equal all the time, because SVG doesn't get into this level of detail. But if we consider all EPUB rendering in terms of html user agent rendering then it works.

Reply:
I've made a change to this effect with the latest commit. Worst case we keep working at massaging this text, I guess... 😄

Reply:
> Can we consider any top-level content document a [top-level browsing context](https://html.spec.whatwg.org/multipage/browsers.html#top-level-browsing-context)?

I worry about that, because in implementations the spine item might not be a top-level browsing context.

Reply:
> because in implementations the spine item might not be a top-level browsing context

But aren't most (all) reading systems using an html core to render content in the spine?

The problem, of course, is that we don't require this to be true, but it's probably true from a practical perspective. Maybe we could note the assumption and that an RS that doesn't use a browser core still needs to restrict access?

Reply:
I've added the following in the latest commit: "If a Reading System does not use a top-level browsing context for Top-level Content Documents, it MUST also prevent data URLs from opening as though they are Top-level Content Documents."

That's the best I can come up with since we lack any alternative name for the rendering context.